WordPress/Classes/Services/CommentService.m

```diff
@@ -730,6 +736,12 @@ - (void)replyToHierarchicalCommentWithID:(NSNumber *)commentID
     // post and content provided.
     BOOL isPrivateSite = post.isPrivate;
     [self createHierarchicalCommentWithContent:content withParent:nil postObjectID:post.objectID siteID:post.siteID completion:^(NSManagedObjectID *commentObjectID) {
+        if (!commentID) {
```

Comment:
This should be `commentObjectID`, correct?

Reply:
Nice catch, thanks. I copy-pasted it from the previous method, but here `commentID` is the ID of the comment you are replying to.